docs/SandboxExtention.md

```diff
@@ -0,0 +1,71 @@
+# OTP Sandbox Extensions
+
+- The sandbox is a place to test and implement new "experimental" features.  
+- This should not be used for bug fixes and smaller changes.
+- Consider forking if the feature is valuable to one deployment only.
+
+
+## Available extensions
+Here is a list of features implemented as OTP Sandbox Extensions. TODO - Disclaimer on usage.
+
+- [Examples](../src/ext/java/org/opentripplanner/ext/examples/README.md) - Sandbox examples on how 
+  to implement extensions.
+
+
+## Terminology
+
+**Main/core**   -- All OTP code and additional files, NOT part of the sandbox. 
+(`docs`, `src/main`, `src/test` and so on)
+
+**Extensions** -- All features implemented in the OTP Sandbox, provided with no guarantees. 
+(`src/ext`, `src/ext-test`)  
+
+## Sandbox Goals
+
+- Reduce work for PR approval
+- Allow experimental code to evolve (in a Sandbox)
+- Encourage refactoring and creation of extension points in the main code.
+- Increase visibility and cooperation of development of new features.
+- Feature toggle 
+  - Default is off
+  - In the config file(s) there should be a list of enabled extensions. To enable an extension add 
+  the name of the extension to the list.
+
+
+## Contract
+- Give your feature a name: `<extension name>`
+- A new feature is isolated from the rest of the code by putting it in the directory `src/ext`. 
+  Java code should have package prefix `org.opentripplanner.ext.<extension name>`. Unit tests 
+  should be added in the test directory: `src/ext-test`
+- To integrate the new feature into OTP you may have to create new extension points in the main 
+  code. 
+- Create a readme file (`src/ext/java/org/opentripplanner/ext/<extension name>/<Extension Name>.md` 
```

Comment:
We will instead put extension documentation on a subpage of the main otp documentation, rather than MD files in source directories.